### PR TITLE
feat(core): fix version of core dependecies

### PR DESCRIPTION
**Pull request by Joelsx**
fix version of core dependecies for continue using actual version of npm

STLX-9878

### CODE REVIEW

File: modules/core/package.json [hunk 1]
```diff
@@ -72,9 +72,9 @@
     "lodash": "^4.17.14",
     "moment": "^2.20.1",
     "ripple-address-codec": "~4.1.3",
-    "ripple-binary-codec": "^0.2.4",
+    "ripple-binary-codec": "~1.1.3",
```
Comment (argjv) [inline modules/core/package.json:75]: Can you try `"ripple-binary-codec": "~0.2.4",` ?

Comment (Joelsx) [reply modules/core/package.json:75]: okay let me try

File: modules/core/package.json [hunk 2]
```diff
@@ -72,9 +72,9 @@
     "lodash": "^4.17.14",
     "moment": "^2.20.1",
     "ripple-address-codec": "~4.1.3",
-    "ripple-binary-codec": "^0.2.4",
+    "ripple-binary-codec": "~0.2.4",
```
Comment (FabianSimtlix) [inline modules/core/package.json:75]: should be ~1.1.3

File: modules/core/package.json (hunk 1 above)
Comment (FabianSimtlix) [reply modules/core/package.json]: Why `0.2.4`? On MS the version currently installed is 1.1.3

Comment (Joelsx) [reply modules/core/package.json]: @argjv Why you want to use the version 0.2.4 since its updating at the moment to the 1.1.3 wich its the stable one?

**Review by tylerlevine — APPROVED** [review]

**Pull request merged**